apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -6,6 +6,7 @@ import Link from "next/link";
 import { usePathname, useRouter } from "next/navigation";
 import type { FC } from "react";
 import { memo, useEffect, useState } from "react";
+import { DragDropContext, Droppable, Draggable } from "react-beautiful-dnd";
```

Comment:
@Amit91848 the unpacked size of this package is 1.39mb. I feel like that's pretty big but I wanted to get other people's opinions.

Reply:
Yeah makes sense. I looked up a few more options lmk what do you think 

1. [Pragmatic Drag and Drop](https://www.npmjs.com/package/@atlaskit/pragmatic-drag-and-drop) from the same guys as React Beautiful Dnd, used at Atlassian for trello jira etc. Unpacked is 407kb
2. [Dnd Kit](https://www.npmjs.com/package/@dnd-kit/core) Unpacked is slightly smaller at 1.05 Mb, but very widely used.
3. [Form kit](https://www.npmjs.com/package/@formkit/drag-and-drop) Unpacked is 516kb but it is only 6-7 months old so not sure how stable it is.
4. I noticed framer-motion is already being used, it has something similar [Reorder](https://www.framer.com/motion/reorder/) .